krizten/Memoire-WebClient
# #165526016 Develop All Entries/Single Entry Page

### What does this PR do?
- Build out all-entries/single entry page.
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the all-entries/single-entry page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ```ft/165526016/all-entries-page```
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/entries```
### What are the relevant pivotal tracker stories?
#165526016
#### Screenshots (if appropriate)
![entries-page-desktop](https://user-images.githubusercontent.com/29978455/57884795-6a0cfa00-7821-11e9-8839-46ec6631b218.png)
![entries-page-mobile](https://user-images.githubusercontent.com/29978455/57884804-72fdcb80-7821-11e9-90b4-8b0b6fd6834c.png)
